src/actions/AppStoreActions.js

```diff
@@ -103,6 +105,12 @@ export const openSnackbar = (variant, message, duration, position, style) => {
 };
 
 export const saveSchedule = async (userID, rememberMe) => {
+    ReactGA4.event({
+        category: analyticsEnum.nav.title,
+        action: analyticsEnum.nav.actions.SAVE_SCHEDULE,
+        label: userID,
+        value: rememberMe,
+    });
```

Comment:
What do you think about exporting a function from `analyticsEnum` that is something like 
```js
function logAnalytics(category, action, label?, value?) {
    ReactGA4.event({
        category: category,
        action: action,
        label: label,
        value: value,
    });
}
```

So that we can just import the function + enum and don't have to import ReactGA4 all over the place?

Reply:
Yeah I agree. implementing this.